packages/components/src/textarea-control/styles/textarea-control-styles.ts

```diff
@@ -2,13 +2,95 @@
  * External dependencies
  */
 import styled from '@emotion/styled';
+import { css } from '@emotion/react';
 
 /**
  * Internal dependencies
  */
-import { inputControl } from '../../utils/input';
+import { font } from '../../utils/font';
+import { COLORS } from '../../utils/colors-values';
+import { CONFIG } from '../../utils';
+import { breakpoint } from '../../utils/breakpoint';
+
+const inputStyleNeutral = css`
+	box-shadow: 0 0 0 transparent;
+	border-radius: ${ CONFIG.radiusSmall };
+	border: ${ CONFIG.borderWidth } solid ${ COLORS.ui.border };
+
+	@media not ( prefers-reduced-motion ) {
+		transition: box-shadow 0.1s linear;
+	}
+`;
+
+const inputStyleFocus = css`
+	border-color: ${ COLORS.theme.accent };
+	box-shadow: 0 0 0
+		calc( ${ CONFIG.borderWidthFocus } - ${ CONFIG.borderWidth } )
+		${ COLORS.theme.accent };
+
+	// Windows High Contrast mode will show this outline, but not the box-shadow.
+	outline: 2px solid transparent;
+`;
+
+const deprecatedStyles = ( { __next40pxDefaultSize = false } ) => {
+	if ( ! __next40pxDefaultSize ) {
+		return css`
+			padding: 6px 8px;
+		`;
+	}
+
+	return css`
+		// Vertical padding is to match the standard 40px control height when rows=1.
+		// Subtracts 1px each to account for the border width.
```

Comment:
Since we're trying to clarify the `9px` value, probably better to mention the `20px` line height?

Reply:
Good call, done 👍